docs/recipes/design-patterns/component-composition.md

```diff
@@ -0,0 +1,133 @@
+# Component composition/extending
+
+Since Skate is already close to the platform, you can easily extend other component classes in 2 ways:
```

Comment:
Maybe remove "in 2 ways", that way it's easier to add without updating `2`, it also feels kinda unnecessary to say.